guillotina_amqp/job.py

```diff
@@ -221,6 +221,7 @@ def state_manager(self):
         # caller
         #
 
+        aiotask_context.set('amqp_job', self)
```

Comment:
out of curiosity: why is this needed?

Reply:
I wanted a way for task to know about itself and this seemed like the most simple mechanism.

I'm open to other ideas!